thirdparty/make-many-rubies/LICENSE

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
```

Comment:
I believe this is inline with the google way of doing things: copy code into a `/thirdparty` dir and then use it. In this case "using" it means copying and modifying from here into the repo root.

Reply:
I'm not sure about this practice for Looker open source projects but I'll take your word for it for now :)

Reply:
ok, I was confused. I have a better approach in mind.

Reply:
Ok, I think I figured this out by looking at the google open source policy for copyright. I copied the original author's file, included his original license/copyright notice, modified the file and added Google LLC to the copyright. got rid of the `/thirdparty` directory